ui/ducks/metamask/metamask.js

```diff
@@ -234,7 +234,7 @@ export const getAlertEnabledness = (state) => state.metamask.alertEnabledness;
  * Get the provider configuration for the current selected network.
  *
  * @param {object} state - Redux state object.
- * @returns {object} The provider configuration for the current selected network.
+ * @returns {import('../../../app/scripts/controllers/network/network-controller').NetworkControllerState['provider']} The provider configuration for the current selected network.
```

Comment:
This was required to fix type errors in TypeScript files where this was referenced; `tsc` was picking up on this type declaration and complaining when trying to access a provider config property.